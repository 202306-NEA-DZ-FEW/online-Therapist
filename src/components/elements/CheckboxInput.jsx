const CheckboxInput = ({ onChange, name, value, text }) => {
    return (
        <div className='flex items-center pl-3'>
            <input
                id='maleCounselor'
                type='checkbox'
                value={value}
                onChange={onChange}
                name={name}
                className='lg:w-4 lg:h-4 text-blue-600 bg-gray-100 border-gray-300 focus:ring-blue-500 focus:ring-2'
            />
            <label
                htmlFor='maleCounselor'
                className='w-full py-1 ml-2 rtl:pr-2'
            >
                {text}
            </label>
        </div>
    );
};

export default CheckboxInput;
