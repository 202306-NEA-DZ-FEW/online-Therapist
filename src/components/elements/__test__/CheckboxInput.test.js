import renderer from "react-test-renderer";

import CheckboxInput from "../CheckboxInput";

it("renders correctly", () => {
    const tree = renderer.create(<CheckboxInput />).toJSON();
    expect(tree).toMatchSnapshot();
});
