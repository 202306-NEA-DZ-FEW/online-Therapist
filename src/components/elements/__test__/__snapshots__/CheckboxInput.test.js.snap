// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex items-center pl-3"
>
  <input
    className="lg:w-4 lg:h-4 text-blue-600 bg-gray-100 border-gray-300 focus:ring-blue-500 focus:ring-2"
    id="maleCounselor"
    type="checkbox"
  />
  <label
    className="w-full py-1 ml-2 rtl:pr-2"
    htmlFor="maleCounselor"
  />
</div>
`;
